docs/processors/builtin.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Built-in processors'
+sidebar_position: 1
+---
+
+:::note
+Note that all built-in processors that operate on the payload actually operate on `Record.Payload.After`.
+If you need to manipulate the field `Record.Payload.Before` you can use
+a [JavaScript processor](./javascript-processors).
+:::
+
+Below is a list of available built-in processors in Conduit. An example usage of a built-in processor is available in
+[`extract-field-transform.sh`](https:github.com/ConduitIO/conduit/blob/main/examples/processors/extract-field-transform.sh).
+The script will run Conduit with a pipeline containing the built-in `extractfieldpayload` processor.
+
+### extractfieldkey
+
+- If the key is raw and has a schema attached, extracts the field and uses it to replace the entire key.
+- If the key is raw and has no schema, returns an error (not supported).
+- If the key is structured, extracts the field and uses it to replace the entire key.
+
+#### Configuration
+
+- `field`
+  (Required) Name of the field which replaces the entire key.
+
+### extractfieldpayload
+
+Works in the same way as [extractfieldkey](#extractfieldkey), except that its applied to `Record.Payload.After`.
+
+### filterfieldkey
+
+`filterfieldkey` includes or excludes records (i.e. drops them from the pipeline) using an XPath expression.
+
+#### Configuration
+
+- `type` (Required)
+  Sets the behavior to "include" or "exclude" the record based on the result of the condition. Possible values
+  are: `include`, `exclude`.
+- `condition` (Required, XPath expression)
+  An XPath query expression that the user defines to forward or drop a record on its results.
+- `missingornull` (Optional)
+  Defines how to handle the record in the event the fields the query would use don't exist. Possible values
+  are: `fail`, `include`, `exclude`. Defaults to `fail`.
+- `exists` (Optional)
+  Gives the user a chance to define an existence query for a given filter. Possible values
+  are: `fail`, `include`, `exclude`. Defaults to `fail`.
```

Comment:
I've copied this from the docs in Conduit, it's a bit unclear, at least to me. Once I'm done with the rest, I'll try add more details and add an example.